CommonProtos/CONTRIBUTING.md

```diff
@@ -0,0 +1,45 @@
+## Contributing
+
+We are pleased that you are interested in contributing to our work.
+
+### Generated Protocol Buffer Classes
+
+The classes in this repository are generated by the protocol buffer
+compiler, as known as protoc. As such, we can not accept contributions
+directly to these generated classes. Instead, changes should be
+suggested upstream in the [Googleapis][googleapis]
```

Comment:
```suggestion
suggested upstream in the [Googleapis][api-common-protos]
```

Reply:
Rather than change here, I changed the reference at the bottom of the file.